README.md

```diff
@@ -177,6 +177,12 @@ spec:
 
 ## Changes
 
+### 0.3.2
```

Comment:
Could you please add information about 0.3.1 version

Reply:
Honestly, I am not sure what should be in that version... So, I suggest we leave it as is